common/src/main/java/com/microsoft/identity/common/nativeauth/internal/commands/AcquireTokenNoFixedScopesCommand.kt

```diff
@@ -59,7 +59,7 @@ class AcquireTokenNoFixedScopesCommand(
 
         Logger.info(
             TAG,
-            "Returning result: $result"
+            "Returning result: AcquireTokenResult()"
```

Comment:
Is there a reason for returning a new result, instead of the one provided from the function above?

Reply:
There's not a new result being returned, it's just a string. So it's not `AcquireTokenResult()` but `"AcquireTokenResult()`.

`infoWithObject()` requires the actual object to implement `ILoggable`. I didn't want to make those changes to `AcquireTokenObject`, as it's owned by MSAL and out of scope of this refactoring. 

So with this code change I actually want to remove the `result` from being printed at all (as it contains data that shouldn't be logged). I wanted to keep something informative there though, so I opted for `"Returning result: AcquireTokenResult()"` to at least inform the developer that this result class is being returned.

Reply:
Ah, I misread. Makes sense 👍